static/app/views/performance/browser/webVitals/utils/getWeights.tsx

```diff
@@ -0,0 +1,33 @@
+import {ProjectScore} from 'sentry/views/performance/browser/webVitals/utils/queries/rawWebVitalsQueries/calculatePerformanceScore';
+
+export const getWeights = (projectScore?: ProjectScore) => {
+  const containsWeights = (
+    weights: ProjectScore
+  ): weights is ProjectScore & {
+    clsWeight: number;
+    fcpWeight: number;
+    fidWeight: number;
+    lcpWeight: number;
+    ttfbWeight: number;
+  } => {
+    return !!(
+      weights?.clsWeight &&
+      weights?.fcpWeight &&
+      weights?.fidWeight &&
+      weights?.lcpWeight &&
+      weights?.ttfbWeight
+    );
+  };
```

Comment:
Same as my comment in the other PR, but this is a bit too defensive given that weights should always be returned instead of omitted. Or if they are a weight of 0 then it would exclude this from returning any weights?

Reply:
Yeah weights should always exist in both backend and frontend scores (hardcoded for frontend). I've just removed this file since I don't really need to guard that it provides anymore